OxfordBodleian/Juel-Jensen/BDLaethf31.xml

```diff
@@ -36,25 +38,101 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     
                     <msContents>
                         <summary/>
-                        
-                        <!--<msItem xml:id="ms_i1">
-                      
-                        </msItem>-->
-                        
+                        <msItem xml:id="ms_i1">
+                            <locus from="3ra" to="75rb"/>
+                            <title type="complete" ref="LIT1693John"/>
+                            <incipit xml:lang="gez">
+                                <locus target="#3r"/>
+                                <hi rend="rubric">ብስራተ፡ አብ፡ ቅዱስ፡ ትሩፍ፡ ረድ<gap reason="illegible" unit="chars" quantity="3"/></hi>ገረ፡ በመለኮት፡ ዮሐንስ፡ ወንጌላዊ፡ 
+                                ወ<hi rend="rubric">ልደ፡ ዘብዴዎስ፡ ፍቁረ፡ እ<hi rend="ligature">ግዚ</hi>እንተ፡ ኢየሱስ፡</hi> ክርስቶስ፡ የሀሉ፡ ምስለ፡ ገብሩ፡ 
+                                <del rend="erasure"/> <hi rend="rubric">በዓለመ፡ ዓለም፡ አሜን።</hi> 
+                            </incipit>
+                            <msItem xml:id="ms_i1.1">
+                                <locus from="3ra" to="75r1"/>
+                                <title type="complete" ref="LIT2715John"/>
+                                <incipit xml:lang="gez">
+                                    <locus from="3ra" to="3rb"/>
+                                    <hi rend="rubric">ቀዳሚሁ፡ ሀለወ፡</hi> ቃል፡ ወውእቱ፡ <hi rend="rubric">ቃል፡ ኀበ፡ እግዚ</hi>አብሔር፡ ውእቱ፡ ወእግዚአብሔር፡ ውእቱ፡ ቃል። 
+                                    ወዝንቱ፡ እምቀዳሚሁ፡ እምቀዲሙ፡ ኀበ፡ እግዚኣብሔር፡ ሀለወ፡ ወኵሉ፡ ቦቱ፡ ኮነ፡ ወዘእንበሌሁሰ፡ <cb n="b"/> <hi rend="rubric">አልቦ፡ ዘኮነ፡ 
+                                        ወኢም</hi>ንትኒ፡ እምዘ፡ ኮ<hi rend="rubric">ነ። ወቦቱ፡ ኮነ፡ ሕይ</hi>ወት።
+                                </incipit>
+                                <explicit xml:lang="gez">
+                                    <locus from="75vb" to="75ra"/>
+                                    ወሶበ፡ ተጽሐፈ፡ ኵሉ፡ በበ፡ አሐዱ፡ እምኢያግመሮ፡ ዓለም፡ ጥቀ፡ መጻሕፍቲሁ። ዘተጽሕፈ፡ አሜን፡ ወአሜን። <pb n="75r"/> ወኵሉ፡ ቃሉ፡ በወንጌለ፡ 
+                                    ጽድቅ፡ ውእቱ፡ እምጥንቱ፡ እስከ፡ ተፍጻሜቱ፡ ለሊሁ፡ ወልድ፡ ተናገረ።
+                                </explicit>
+                            </msItem>
+                            <msItem xml:id="ms_i1.2">
+                                <locus from="75ra" to="75rb"/>
+                                <title type="complete" ref="LIT1693John#Postscript"/>
+                                <note> The subscription is separated from the explicit of the text by means of two chains of black and red dots. 
+                                    <q xml:lang="gez">
+                                    <hi rend="rubric">መልአ፡ ጽሕፈተ፡ ብስራቱ፡ ለዮሐንስ፡ ዜናዊ።</hi> ወልደ፡ ዘብዴዎስ፡ ሐዋርያ፡ ፩እም፲ወ፪ሰባክያን። ወጸሐፋ፡ እንከ፡ በልሳነ፡ 
```

Comment:
ጸሐፈ instead of ጸሐፋ?

Reply:
that's right ("he wrote-it", i.e. the Gospel, taken as a feminine noun)